modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -19,20 +20,23 @@ For this use case Push Notifications are recommended.
 To enable subscriptions on your client set the following
 paramater in the {sync-client} config object. A DataSyncConfig interface is also available from {sync-client} if you wish to use it.
 
-== Setting up a Client to use Subscriptions
+== Setting up a client to use subscriptions
 
-To set up a client to use subscriptions simply provide a wsUrl string in the config object.
+To set up a client to use subscriptions:
 
+. Provide a `wsUrl` string in the config object as follows:
++
 [source,javascript]
 ----
 const config = {
-    wsUrl: "ws://your_websocket_url"
+    <wsUrl>: "ws://your_websocket_url"
```

Comment:
We want user to replace the **value** of the object's property here, i.e. `ws://your_websocket_url`, not `wsUrl`. So would replacing:
`<wsUrl>: "ws://your_websocket_url"` for `wsUrl: "<websocket_url>"` make more sense?

If so, let's replace `<wsUrl>` for `<websocket_url>` in the following sentence as well

Reply:
doh! yep, that's what i was trying to do